src/content/en/updates/2018/06/feature-policy.md

```diff
@@ -0,0 +1,336 @@
+{# wf_blink_components: Blink>FeaturePolicy #}
+
+# Introduction to Feature Policy {: .page-title }
+
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+<style>
+figure {
+  text-align: center;
+}
+figcaption {
+  font-style: italic;
+}
+</style>
+
+#### TL;DR {: #tldr .hide-from-toc}
+
+[Feature Policy][spec] allows web developers to selectively enable, disable, and
+modify the behavior of certain APIs and web features in the browser. __It's like
+[CSP](/web/fundamentals/security/csp/) but instead of controlling security, it's
+controls features!__
+
+The feature policies themselves are little opt-in aggreements between developer
+and browser that can help foster our goals of building (and maintaining) high
+quality web apps.
+
+## Introduction {: #intro}
+
+Building for the web is a rocky adventure. It's hard enough to build a top-notch
+web app that nails performance and uses all the latest best practices. It's even
+harder to keep that experience great over time. As your project evolves,
+developers come on board, new features land, and the codebase grows. That
+Great Experience &trade; you once achieved may begin to deteriorate and UX
+starts to suffer! Feature Policy is designed to keep us on track.
+
+With Feature Policy, you **opt-in to a set of "policies"** for the browser to
+enforce on specific features used throughout your site. These policies can
+restrict what APIs the site can access or modify the browser's default
+behavior for certain features.
+
+Here are a few examples of the types of things you can do with Feature Policy:
+
+1. Change the default behavior of `autoplay` on mobile and 3rd party videos.
+2. Ensure a site doesn't inadvertently use sensitive APIs like `camera` or
+`microphone`.
+3. Allow iframes to use the `fullscreen` API.
+4. Block the use of outdated APIs like synchronous XHR and `document.write`.
+5. Ensure images are sized properly (e.g. prevent layout thrashing) and are not
+too big for the viewport (e.g. waste user's bandwidth).
+
+**Policies are a contract between developer and browser**. They inform the
+browser about what the developer's intent is and thus, help keep us honest when
+our app tries to go off the rails and do something bad. If the site or embedded third-party content attempts to violate any of the developer's preselected
+rules, the browser overrides the behavior with better UX or blocks the API
+altogether.
+
+## Using Feature Policy {: #using }
+
+Feature Policy provides two ways to control features:
+
+1. Through the `FeaturePolicy` HTTP header.
+2. With the `allow` attribute on iframes.
+
+### The `FeaturePolicy` HTTP header {: #header }
+
+The easiest way to use Feature Policy is by sending the `FeaturePolicy` HTTP
+header with the response of a page. It's value is the policy or set of
+policies that you want the browser to respect for a given origin:
+
+```
+FeaturePolicy: <feature> <allow list origin(s)>
```

Comment:
Double-check that '<' doesn't need to be escaped to prevent it from swallowing the rest of the page.